gateway/src/apicast/version.lua

```diff
@@ -1 +1 @@
-return "3.8.0-alpha2"
+return "3.8.0-rc1"
```

Comment:
rc1 releases can become GA, that's why in the past we've omitted this part.

Reply:
Would be good if we work towards consequent naming between various components. (i.e. https://github.com/3scale/porta/releases vs https://github.com/3scale/APIcast/releases)

Reply:
@thomasmaas You mean not having upstream version numbers and only product ones?

Reply:
No, I mean using "alpa2", lowercase, etc